CMakeLists.txt

```diff
@@ -103,6 +105,81 @@ if(UNIX AND
     CMAKE_INSTALL_PREFIX.")
 endif()
 
+# Configure installation directories across platforms.
+# ----------------------------------------------------
+
+## OPENSIM_INSTALL_UNIX_FHS option.
+set(OPENSIM_INSTALL_UNIX_FHS_DEFAULT OFF)
```

Comment:
My understanding about this option is:
* When ON, the installation layout is what unix uses in its standard directories. This option *has to be ON* for a user to install OpenSim in standard directories.
* When OFF, the installation layout is not the one unix uses. This can be used while installing OpenSim anywhere but standard directories. 

I am suggesting to have this option *ON by default* resulting in OpenSim installation layout being the same all the time -- like unix standard directories. This way user can install anywhere in unix and the layout will look the same. User is allowed to turn this option OFF by going advanced on CMake but will see a fatal error if attempting to install in unix standard directories.